test/behavior/nesting.ts

```diff
@@ -248,37 +248,47 @@ async function shouldBehaveLikeNesting(
     });
 
     it('can accept child', async function () {
-      await expect(parent.connect(tokenOwner).acceptChild(parentId, 0))
+      await expect(parent.connect(tokenOwner).acceptChild(parentId, 0, child.address, childId))
         .to.emit(parent, 'ChildAccepted')
-        .withArgs(parentId, child.address, childId, 0);
+        .withArgs(parentId, 0, child.address, childId);
       await checkChildWasAccepted();
     });
 
     it('can accept child if approved', async function () {
       const approved = addrs[1];
       await parent.connect(tokenOwner).approve(approved.address, parentId);
-      await parent.connect(approved).acceptChild(parentId, 0);
+      await parent.connect(approved).acceptChild(parentId, 0, child.address, childId);
       await checkChildWasAccepted();
     });
 
     it('can accept child if approved for all', async function () {
       const operator = addrs[2];
       await parent.connect(tokenOwner).setApprovalForAll(operator.address, true);
-      await parent.connect(operator).acceptChild(parentId, 0);
+      await parent.connect(operator).acceptChild(parentId, 0, child.address, childId);
       await checkChildWasAccepted();
     });
 
     it('cannot accept not owned child', async function () {
       const nowOwner = addrs[3];
-      await expect(parent.connect(nowOwner).acceptChild(parentId, 0)).to.be.revertedWithCustomError(
-        parent,
-        'ERC721NotApprovedOrOwner',
-      );
+      await expect(
+        parent.connect(nowOwner).acceptChild(parentId, 0, child.address, childId),
```

Comment:
Is `nowOwner` a typo or should it be called that?

Reply:
It was a typo, thanks